Templates/templates/CSharp/Base/ICollectionRequest.Base.template.tt

```diff
@@ -33,18 +33,25 @@ public string GetCollectionWithReferencesInterfaceDefinition(OdcmProperty odcmPr
 // -------------------------------------------------------------
 // Build CRUD methods
 // -------------------------------------------------------------
-public void AppendMethodHeaderToPostAsyncMethod(string propertyType, string sanitizedPropertyName, StringBuilder stringBuilder)
+public void AppendMethodHeaderToPostAsyncMethod(string propertyType, string sanitizedPropertyName, StringBuilder stringBuilder, bool isGraphResponseMethod)
```

Comment:
can we have everything that calls stringBuilder.Append etc moved into a CS file please? for maintainability and ease of review.

Reply:
it just makes it hard to review for us right now. I'd like things to be moved, but if this is too much of a burden leave it. Your call @andrueastman 😊

Reply:
I will leave it for now.